src/locale/locales/pt-BR/messages.po

```diff
@@ -4835,7 +4835,7 @@ msgstr "Tocar ou pausar o GIF"
 #: src/view/com/util/post-embeds/VideoEmbed.tsx:52
 #: src/view/com/util/post-embeds/VideoEmbedInner/VideoWebControls.tsx:204
 msgid "Play video"
-msgstr ""
+msgstr "Tocar vídeo"
```

Comment:
"Reproduzir" is usually used in this context

Reply:
ok

Reply:
ok